pkg/operator/backupcontroller/backupcontroller.go

```diff
@@ -0,0 +1,257 @@
+		kubeInformersForOpenshiftEtcdNamespace.Core().V1().ConfigMaps().Informer(),
+		kubeInformersForOpenshiftEtcdNamespace.Core().V1().Secrets().Informer(),
+		kubeInformersForNamespaces.InformersFor("").Core().V1().Nodes().Informer(),
+		infrastructureInformer.Informer(),
+		networkInformer.Informer(),
+	).WithSync(c.sync).ToController("BackupConfigController", syncCtx.Recorder())
+}
+
+func (c BackupConfigController) sync(ctx context.Context, syncCtx factory.SyncContext) error {
+	operatorSpec, _, _, err := c.operatorClient.GetStaticPodOperatorState()
+	if err != nil {
+		return err
+	}
+	requeue, err := createBackupConfig(c, syncCtx.Recorder(), operatorSpec)
+	if err != nil {
+		return err
+	}
+	if requeue {
+		return fmt.Errorf("synthetic requeue request")
+	}
+
+	return nil
+}
+
+func createBackupConfig(c BackupConfigController, recorder events.Recorder, operatorSpec *operatorv1.StaticPodOperatorSpec) (bool, error) {
+	errors := []error{}
+
+	contentReplacer, err := c.getSubstitutionReplacer(operatorSpec, c.operatorImagePullSpec, c.operatorImagePullSpec)
+	if err != nil {
+		return false, err
+	}
+
+	_, _, err = c.manageBackupPod(contentReplacer, c.kubeClient.CoreV1(), recorder, operatorSpec)
+	if err != nil {
+		errors = append(errors, fmt.Errorf("%q: %v", "configmap/etcd-pod", err))
+	}
+
+	if len(errors) > 0 {
+		condition := operatorv1.OperatorCondition{
+			Type:    "BackupConfigControllerDegraded",
+			Status:  operatorv1.ConditionTrue,
+			Reason:  "SynchronizationError",
+			Message: v1helpers.NewMultiLineAggregate(errors).Error(),
+		}
+		if _, _, err := v1helpers.UpdateStaticPodStatus(c.operatorClient, v1helpers.UpdateStaticPodConditionFn(condition)); err != nil {
+			return true, err
+		}
+		return true, nil
+	}
+
+	condition := operatorv1.OperatorCondition{
+		Type:   "BackupConfigControllerDegraded",
+		Status: operatorv1.ConditionFalse,
+	}
+	if _, _, err := v1helpers.UpdateStaticPodStatus(c.operatorClient, v1helpers.UpdateStaticPodConditionFn(condition)); err != nil {
+		return true, err
+	}
+
+	return false, nil
+}
+
+func loglevelToKlog(logLevel operatorv1.LogLevel) string {
+	switch logLevel {
+	case operatorv1.Normal:
+		return "2"
+	case operatorv1.Debug:
+		return "4"
+	case operatorv1.Trace:
+		return "6"
+	case operatorv1.TraceAll:
+		return "8"
+	default:
+		return "2"
+	}
+}
+
+func (c *BackupConfigController) getSubstitutionReplacer(operatorSpec *operatorv1.StaticPodOperatorSpec, imagePullSpec, operatorImagePullSpec string) (*strings.Replacer, error) {
+	hostResourceDirDir := "/etc/kubernetes/static-pod-resources"
+	hostPodManifestDir := "/etc/kubernetes/manifests"
+
+	args := []string{
+		fmt.Sprintf("-v=%d", 2),
+		fmt.Sprintf("--revision=%d", 2),
+		fmt.Sprintf("--namespace=%s", "openshift-etcd"),
+		fmt.Sprintf("--pod=%s", "backup-pod"),
+		fmt.Sprintf("--resource-dir=%s", hostResourceDirDir),
+		fmt.Sprintf("--pod-manifest-dir=%s", hostPodManifestDir),
```

Comment:
look into dropping this as we don't need the manifest dir created by the installer.